modules/apps/foundation/frontend-theme/frontend-theme-user-dashboard/src/templates/portal_normal.ftl

```diff
@@ -52,15 +52,11 @@
 					</#if>
 				</div>
 
-				<#assign VOID = freeMarkerPortletPreferences.setValue("portletSetupPortletDecoratorId", "barebone") />
```

Comment:
@marcoscv-work,

we're just trying to understand the current design.  Are these lines needed by the taglib?